### DISCUSSION TITLE
Installing Github Copilot Chat 0.23.2024102903

### DISCUSSION THREAD

**Discussion by cycl0ne** [coder/code-server#7081] [Q&A] [answered]
Hi,
maybe someone can help, i got a local vscode and i installed code-server on a virtual machine in the net.
localy i use 1.95.2
Net i use 1.95.3 - 4.95.3

i installed it localy without problem: 0.23.2024102903

i download the github tools (chat and non chat) to install it on the remote server. i can install the normal github copilot, but the chat fails. 
looking into manifest of chat, i find this:

Property Id="Microsoft.VisualStudio.Code.Engine" Value="^1.96.0-20241114" 

but i wonder, since it runs localy on my 1.95.2 machine. Any ideas?

**Comment by code-asher** [comment]
You will need to install an older version of the extension that is compatible with 1.95.2.

**Reply by cycl0ne** [reply]
yes i did that, but i was wondering, why in the local installmant of visual studio (1.95,2) this plugin works. M$ publish only old version on their store with  Value="^1.96.0, i had to search inet for a older version

**Reply by code-asher** [reply]
Are you sure it is the same extension?  If you installed it from the sidebar, then it automatically will install the right version (which is sometimes an older one).  If you installed the 1.96.0 one manually then I am not sure how it would work, unless you are running the insiders version of VS Code?

**Reply by cycl0ne** [reply, marked as answer]
see this is what i also find astonishing, here my version of visualcode:
![image](https://github.com/user-attachments/assets/c1df6d00-baaf-464d-8d45-f5ef58f3dc8c)

and here the copilot version, im on the prerelease channel:
![image](https://github.com/user-attachments/assets/1e7a093d-b184-4d3b-8adc-c33c3f41c79b)

starting from 0.23. all are with the 1.96 check. BUT my version also a 0.23 (not available for normal access in store, what i checked) has the : 
<Property Id="Microsoft.VisualStudio.Code.Engine" Value="^1.95.0-20241022"

i got it by using this url:
https://marketplace.visualstudio.com/_apis/public/gallery/publishers/GitHub/vsextensions/copilot-chat/0.23.2024102903/vspackage

so yes, thanks :-) learned a lot more (and a lot more on how to get what you want) ! :-D

AND THANK YOU FOR YOUR SUPPORT AND THE REALLY COOL CODE-SERVER ! :-)

**Reply by code-asher** [reply]
That makes sense!
